### DISCUSSION TITLE
Disable onboardin survey.

### DISCUSSION THREAD

**Discussion by IDDQD69** [uxbox/uxbox#4309] [General]
Is there a way to disable survey after fresh login?

**Comment by schelmo** [comment]
i think you can do
PENPOT_FLAGS=enable-registration enable-login-with-password **disable-onboarding**
in https://github.com/penpot/penpot/blob/83f8218bf3e23fc74b7a4a37914838f5976d76f2/docker/images/docker-compose.yaml#L96C9-L96C68
